libs/partners/openai/tests/integration_tests/chat_models/test_base.py

```diff
@@ -563,6 +572,21 @@ def test_manual_tool_call_msg() -> None:
         llm_with_tool.invoke(msgs)
 
 
+def test_bind_tools_tool_choice() -> None:
+    """Test passing in manually construct tool call message."""
+    llm = ChatOpenAI(model="gpt-3.5-turbo-0125", temperature=0)
+    for tool_choice in ("any", "required"):
```

Comment:
why not test the other possible values?

Reply:
this is specifically testing that a tool is called even when the prompt is irrelevant to any tool